src/other/Slider.css

```diff
@@ -35,3 +35,28 @@
     stroke-width: 1.00px;
     pointer-events: none;
 }
+
+.common_Icon:hover {
```

Comment:
Actually they are only meant to work on the icon here, I don't want to override the default icon settings, in the main icon styles file.